test/action.test.js

```diff
@@ -150,9 +150,7 @@ tap.test('should skip PR with non dependabot commit', async () => {
 
   stubs.prCommitsStub.resolves([
     {
-      author: {
-        login: 'not dependabot',
-      },
+      author: undefined,
```

Comment:
we need to test both cases, not just once. to avoid duplication of the whole test, just create an array with the two users and create the 2 tests in a loop